docs/extensions.rst

```diff
@@ -30,6 +30,7 @@ explicitly invite it (e.g. everything by default is *Private*). These currently
 include (see the docstrings in the specific components for details):
```

Comment:
I know this wasn't changed as part of this PR, but would you mind clarifying the "These" in "These currently include" to specify whether these **should** or **should not** be modified?

Also - if you are indicating that *only* haas.auth is available as an interface and not children, should that be explicit?